src/main/java/uk/gov/hmcts/reform/preapi/dto/CaseDTO.java

```diff
@@ -36,6 +38,12 @@ public class CaseDTO {
     @Schema(description = "CaseIsTest")
     private boolean test;
 
+    @Schema(description = "CaseState")
+    private CaseState state = CaseState.OPEN; 
```

Comment:
```suggestion
    private CaseState state;
```

You also need to add these lines of code to the constructor:
```java
state = caseEntity.getState();
closedAt = caseEntity.getClosedAt();
```